### PR TITLE
364 adapt text notification

### PR DESCRIPTION
**Pull request by bscheltinga**
Fixes: https://github.com/PerfectFit-project/testing-tickets/issues/13
Fixes: https://github.com/PerfectFit-project/virtual-coach-issues/issues/364

### CODE REVIEW

File: Rasa_Bot/domain/domain_notifications.yml [hunk 1]
```diff
@@ -26,23 +26,43 @@ slots:
...
+             van het Perfect Fit programma, die je aan het begin hebt gekregen, staat uitgelegd hoe je dit kunt
+             doen. 📋"
+    - text: "Je kunt (aan het einde van de dag) in de app te registreren hoeveel sigaretten je hebt gerookt. In de
+            handleiding van het Perfect Fit programma, die je aan het begin hebt ontvangen, vind je instructies over hoe
+            je dit kunt doen. 📋"
   utter_notifications_tracking_3:
     - text: "Vergeet ook niet om je horloge weer om te doen. ⌚ Ben je benieuwd hoe actief je vandaag bent? Dan hoef je
             alleen maar op je horloge te kijken!"
+    - text: "Vergeet niet om je horloge weer om te doen. ⌚ Als je wilt weten hoe actief je vandaag bent geweest, kun je
+             op je horloge kijken! "
```
Comment (nelealbers) [inline Rasa_Bot/domain/domain_notifications.yml:44]:
```suggestion
             op je horloge kijken!"
```

File: Rasa_Bot/domain/domain_notifications.yml [hunk 2]
```diff
@@ -57,24 +77,37 @@ responses:
...
+    - text: "Alvast veel succes! 🍀"
 
 # Step goal and intensive pa notification
   utter_notifications_step_goal_intensive_pa_1:
     - text: "Deze week is je doel om elke dag 8000 stappen te zetten 🚶🏿 en minimaal {notifications_pa_intensity_goal}
              minuten intensief te bewegen. 🏊 🚴 🤸"
+    - text: "Je doel voor deze week is om elke dag 8000 stappen te zetten 🚶🏿 en ten minste
+             {notifications_pa_intensity_goal}  minuten intensief te bewegen 🏊 🚴 🤸"
+    - text: "Het weekdoel is om dagelijks 8000 stappen te zetten 🚶🏿 en ten minste
+             {notifications_pa_intensity_goal} minuten intensief te bewegen 🏊 🚴🤸 "
```
Comment (nelealbers) [inline Rasa_Bot/domain/domain_notifications.yml:89]:
```suggestion
             {notifications_pa_intensity_goal} minuten intensief te bewegen 🏊 🚴🤸"
```

**Review by nelealbers — APPROVED** [review]
Looks good to me! I just removed some spaces at the end because I always find those confusing when writing tests.

And yes, I agree that we should update the changes in the excel files on Teams. Maybe not in red (because that suggests that the changes are not incorporated yet) though, but some other color.

Also, good catch that the alternative formulations were not there yet.

Lastly, I don't know if there are any tests with these notifications, but if yes, would be good to update those too.

File: Rasa_Bot/domain/domain_notifications.yml (hunk 2 above)
Comment (nelealbers) [reply Rasa_Bot/domain/domain_notifications.yml:89]: Actually, there is a period missing in the above statement too now that I look at it.

**Comment by bscheltinga** [comment]
Thanks @nelealbers, I updated the excel filesl

**Pull request merged**